_about/projects/wai-guide/index.md

```diff
@@ -37,7 +37,79 @@ This page provides information on the Web Accessibility Initiative - Authoritati
 -   W3C Group Notes for [XR Accessibility User Requirements](https://www.w3.org/TR/xaur/), [RTC Accessibility User Requirements](https://www.w3.org/TR/raur/), and [Synchronization Accessibility User Requirements](https://www.w3.org/TR/saur/), to help address accessibility of immersive environments , real-time communication, and audio and visual media synchronization
 -   On-going development of [ATAG Report Tool (ART)](https://www.w3.org/WAI/atag/report-tool/) to help promote accessibility of authoring tools
 -   Launch of [Online Course: Introduction to Web Accessibility](https://www.w3.org/blog/2019/12/free-online-course-introduction-to-web-accessibility/) based on the curriculum developed through WAI-Guide
--   Release of [WAI Curricula](https://www.w3.org/WAI/curricula/) with foundation, developer, and designer modules. ([Content Author Modules(https://wai-curricula.netlify.app/curricula/content-author-modules/) in development)
+-   Release of [WAI Curricula](https://www.w3.org/WAI/curricula/) with foundation, developer, and designer modules. [Content Author Modules](https://wai-curricula.netlify.app/curricula/content-author-modules/) in development)
```

Comment:
```suggestion
-   Release of [WAI Curricula](https://www.w3.org/WAI/curricula/) with foundation, developer, designer, and content author modules.
```

Reply:
This one relates to previous announcements. At the time of the announcement (which admittedly isn't included in the listing) the Content Author Modules were not released. On that basis, I probably wouldn't include this suggestion.

Having said all that we may end up just removing all the announcements!